spec/system/visit_msa_component_new_spec.rb

```diff
@@ -53,7 +53,22 @@
       choose('component_environment_staging')
       click_button 'Create MSA component'
 
-      expect(page).to have_content t('components.errors.invalid_entity_id')
+      expect(page).to have_content t('components.errors.existing_entity_id')
+    end
+  end
+
+  context 'creation fails with whitespaces within entity id' do
+    it 'when entity id in a msa component has spaces' do
+      entity = ' test entity id '
```

Comment:
Same comment here - test stripping and failing on whitespace in the middle in two separate tests